src/data/auroradb.ts

```diff
@@ -0,0 +1,251 @@
+import AWS from 'aws-sdk';
+import dateFormat from 'dateformat';
+import { createPool } from 'mysql2/promise';
+
+export interface ServiceType {
+    lineName: string;
+    startDate: string;
+    description: string;
+}
+
+export interface JourneyPattern {
+    startPoint: {
+        Id: string;
+        Display: string;
+    };
+    endPoint: {
+        Id: string;
+        Display: string;
+    };
+    stopList: string[];
+}
+
+export interface QueryData {
+    operatorShortName: string;
+    serviceDescription: string;
+    lineName: string;
+    fromAtcoCode: string;
+    toAtcoCode: string;
+    fromCommonName: string;
+    toCommonName: string;
+    journeyPatternSectionId: string;
+    order: string;
+}
+export interface RawJourneyPatternSection {
+    OrderedStopPoints: {
+        StopPointRef: string;
+        CommonName: string;
+    }[];
+}
+
+export interface RawJourneyPattern {
+    JourneyPatternSections: RawJourneyPatternSection[];
+}
+
+interface NaptanInfo {
+    commonName: string;
+    naptanCode: string;
+    atcoCode: string;
+    nptgLocalityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator: string;
+    street: string;
+}
+
+interface NaptanAtcoCodes {
+    naptanCode: string;
+    atcoCode: string;
+}
+
+export interface Stop {
+    stopName: string;
+    naptanCode: string;
+    atcoCode: string;
+    localityCode: string;
+    localityName: string;
+    parentLocalityName: string;
+    indicator?: string;
+    street?: string;
+    qualifierName?: string;
+}
+
+export interface StopIdentifiers {
+    naptanCode: string | null;
+    atcoCode: string;
+}
+
+export interface Service {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: JourneyPattern[];
+}
+
+export interface RawService {
+    serviceDescription: string;
+    operatorShortName: string;
+    journeyPatterns: RawJourneyPattern[];
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const getAuroraDBClient = () => {
+    let client = null;
+
+    if (process.env.NODE_ENV === 'development') {
+        client = createPool({
+            host: 'localhost',
+            user: 'fdbt_site',
+            password: 'password',
+            database: 'fdbt',
+            waitForConnections: true,
+            connectionLimit: 10,
+            queueLimit: 0,
+        });
+    } else {
+        const ssm = new AWS.SSM();
+        client = createPool({
+            host: process.env.RDS_HOST,
+            user: ssm.getParameter({ Name: 'fdbt-rds-site-username', WithDecryption: true }).toString(),
```

Comment:
The value for a parameter needs to be accessed using `ssm.getParameter({ Name: 'fdbt-rds-site-username', WithDecryption: true })['Parameter']['Value']`